src/openzaak/templates/maykin_2fa/base.html

```diff
@@ -0,0 +1,11 @@
+{% extends "maykin_2fa/base.html" %}
+{% comment %} SPDX-License-Identifier: EUPL-1.2 {% endcomment %}
+{% comment %} Copyright (C) 2019 - 2020 Dimpact {% endcomment %}
+
+{# Django 3.2 #}
+{% block breadcrumbs %}{% endblock %}
+
+{# Do not show any version information #}
+{% block footer %}
+    <div id="footer"></div>
+{% endblock %}
```

Comment:
Same comment as for ON, this file can be removed

Reply:
Sorry, I was wrong about removing the `base.html` file, it turns out Open Zaak does use this `footer` block https://github.com/open-zaak/open-zaak/blob/e701f8155735949033fc53f0c78ac4ce9fcc97ca/src/openzaak/templates/admin/base_site.html#L43 (while Open Notificaties does not). Could you re-add it here?